app/controllers/meetings_controller.rb

```diff
@@ -0,0 +1,60 @@
+class MessagesController < ApplicationController
+  before_action :set_message, only: %i[show update destroy]
+
+  # GET /messages
+  def index
+    @messages = Message.all
+
+    render json: @messages
+  end
+
+  # GET /messages/1
+  def show
+    render json: @message
+  end
+
+  # POST /messages
+  def create
+    @message = Message.new(message_params)
+
+    if @message.save
+      render json: @message, status: :created, location: @message
+    else
+      render json: @message.errors, status: :unprocessable_entity
+    end
+  end
+
+  # PATCH/PUT /messages/1
+  def update
+    if @message.update(message_params)
+      render json: @message
+    else
+      render json: @message.errors, status: :unprocessable_entity
+    end
+  end
+
+  # DELETE /messages/1
+  def destroy
+    @message.destroy
+  end
+
+  def random
+    @message = Message.order('RANDOM()').first
+    render json: @message
+
+    # random_greeting = Message.order('RANDOM()').first
+    # render json: { greeting: random_greeting.content }
+  end
```

Comment:
- [ ] Hi @unorjikingsley, random greetings are not still displayed after creating different greetings through rails c. I got the error on the screenshot below.
![Screenshot from 2023-09-19 20-05-32](https://github.com/unorjikingsley/hello-rails-back-end/assets/63087491/a6e593c0-b512-4ed3-84b6-816eed0b615f)